acct_mgt_test.py

```diff
@@ -477,8 +467,17 @@ def test_project(acct_mgt_url, auth_opts):
 
 
 def test_user(acct_mgt_url, auth_opts):
-    # if(oc_resource_exist("user", "test01",r'test01[ \t]*[a-f0-9\-]*[ \t]*sso_auth:test01',r'Error from server (NotFound): users.user.openshift.io "test01" not found')):
-    #    print("Error: test_user failed as a user with a name of test01 exists.  Please delete first and rerun the tests\n")
+    """This tests user (and user identity) existence/creation/deletion"""
+    # if oc_resource_exist(
+    #    "user",
+    #    "test01",
+    #    r"test01[ \t]*[a-f0-9\-]*[ \t]*sso_auth:test01",
+    #    r'Error from server (NotFound): users.user.openshift.io "test01" not found',
+    # ):
+    #    print(
+    #        "Error: test_user failed as a user with a name of test01 exists.\n" +
+    #        "Please delete first and rerun the tests\n"
+    #    )
```

Comment:
why was this test commented out to begin with?

Reply:
I am still considering if I want that or not.  What it told me was that the test didn't complete successfully the last time I ran the tests - something I already knew!  Doesn't seem to be the most useful test.  

I like it because it prevents the test from overwriting the hapless user who purposely has an account named test01 (perhaps shortened from "Tom Tesla").  Yeah, I know, contrived.

Without that check in the beginning of the test, what happens is that it falls though and the test proceeds normally and is cleaned up without me having to go in and delete 5 projects, 5 users and 5 identities in openshift.  Greatly decreases the development time as I didn't have to have a separate script go though and delete the testing left overs.